test/login-web-ui-test.js

```diff
@@ -52,34 +52,33 @@ suite.addBatch({
             },
             "and we visit the login URL": {
                 topic: function() {
-                    var browser;
+                    var cb = this.callback;
                     browser = new Browser({runScripts: true});
```

Comment:
Should be `var browser` or the previous line should end in a comma and this line should be indented